pkgs/development/python-modules/rpy2/default.nix

```diff
@@ -58,6 +58,10 @@ buildPythonPackage rec {
       tidyr
     ]) ++ extraRPackages ++ rWrapper.recommendedPackages;
 
+    nativeBuildInputs = [
+      R # needed at setup time to detect R_HOME (alternatively set R_HOME explicitly)
```

Comment:
@matthewbauer what is the right way to do this? This is probably wrong right? It would pick up the build `R` and not be able to find it at runtime.